core/src/main/kotlin/com/exactpro/th2/read/db/core/impl/DataBaseMonitorServiceImpl.kt

```diff
@@ -101,7 +101,9 @@ class DataBaseMonitorServiceImpl(
             dataSourceId,
             initQueryId,
             initParameters,
-        ).lastOrNull()
+        ).onEach {
+            updateListener.onUpdate(dataSourceId, it)
+        }.lastOrNull()
```

Comment:
This is not compatible with the old behavior. Believe we should introduce a parameter for the task and depending on the parameter's value call this method or don't

Reply:
Maybe I wrong and we can configure base read-db version to start pulling from empty init query result